runtime/compiler/env/j9methodServer.hpp

```diff
@@ -274,6 +301,7 @@ class TR_ResolvedRelocatableJ9JITServerMethod : public TR_ResolvedJ9JITServerMet
    virtual void                  handleUnresolvedStaticMethodInCP(int32_t cpIndex, bool * unresolvedInCP) override;
    virtual void                  handleUnresolvedSpecialMethodInCP(int32_t cpIndex, bool * unresolvedInCP) override;
    virtual void                  handleUnresolvedVirtualMethodInCP(int32_t cpIndex, bool * unresolvedInCP) override;
+   virtual char *                fieldOrStaticNameChars(int32_t cpIndex, int32_t & len) override;
```

Comment:
This method is something I removed from jitaas and should be removed from master branch as well.

Reply:
Will remove it. I thought it was the other way around.